lib/c_glib/src/thrift/c_glib/transport/thrift_ssl_socket.c

```diff
@@ -175,15 +179,15 @@ void thrift_ssl_socket_get_ssl_error(ThriftSSLSocket *socket, const guchar *erro
 	  break;
 	case SSL_ERROR_SYSCALL:
 	  buffer_size-=snprintf(buffer, buffer_size, "%s: ", error_msg);
-	  buffer_size-=snprintf(buffer+(1024-buffer_size), buffer_size, "%lX -> %s", errno, strerror(errno));
+	  buffer_size-=snprintf(buffer+(1024-buffer_size), buffer_size, "%X -> %s", errno, strerror(errno));
 	  break;
```

Comment:
Mind to explain this change? What warning did you get here and why does this solve it?

Reply:
@Jens-G because `errno` is an `int` rather than `long int`,  the warning is :  
`src/thrift/c_glib/transport/thrift_ssl_socket.c:178:69: warning: format ‘%lX’ expects argument of type ‘long unsigned int’, but argument 4 has type ‘int’ [-Wformat=]
    buffer_size-=snprintf(buffer+(1024-buffer_size), buffer_size, "%lX -> %s", errno, strerror(errno));`

Reply:
I see, thanks.